pkg/management/component.go

```diff
@@ -302,11 +304,20 @@ func (c *component) GetRequiredActions(ctx context.Context, realmName string) ([
 	return requiredActionsRep, nil
 }
 
-func (c *component) CreateUser(ctx context.Context, realmName string, user api.UserRepresentation, generateUsername bool) (string, error) {
+func (c *component) CreateUser(ctx context.Context, realmName string, user api.UserRepresentation, generateUsername bool, generateNameID bool) (string, error) {
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 	var ctxRealm = ctx.Value(cs.CtContextRealm).(string)
 
 	var userRep = api.ConvertToKCUser(user)
+	if generateNameID {
+		if oneUUID, errUUID := uuid.NewUUID(); errUUID!=nil {
+			c.logger.Warn(ctx, "msg", "Can't generate UUID", "err", errUUID.Error())
+			return "", errUUID
+		} else {
```

Comment:
no need to put it in else {}

Reply:
With this syntax ( := in a if statement), affected variables have a scope limited to the if itself. If you don't write it in a else block, you're out of scope and can't use oneUUID anymore.

Reply:
golint also doesn't like it... 2 voices for you, one for me... I'll make a change! ;-)